### DISCUSSION TITLE
Continuous integration (CI) for FATES regression testing and HLM compatibility testing

### DISCUSSION THREAD

**Discussion by glemieux** [NGEET/fates#1162] [Ideas]
This discussion point is mainly to note that E3SM is starting to develop CI testing for pull requests: https://github.com/E3SM-Project/E3SM/pull/6231.  If their container development and workflow provides for officially supported e3sm containers, we may be able to adopt these for our usage in automating some fates regression testing.  I could see having automated build tests upon PR creation and automated regression testing of fates-specific testmods upon approval as being helpful in streamlining development workload.  

It might also be helpful to run weekly build and regression tests using the latest fates tag with the latest HLM commit/tag to catch potential incompatibilities with the run modes that aren't tested as part of the default HLM test list (e.g. `e3sm_land_developer`).

**Comment by ekluzek** [comment]
Thanks for pointing this out @glemieux this would be great to keep tabs on and adopt as we are able to...
